docs/rules/device-details/npn_05v0/npn_05v0-table0.rst

```diff
@@ -15,82 +15,82 @@
      - 18.14
      - 56.93
      - 
-     - NPN forward Current Gain (I\ :sub:`C`/I\ :sub:`B`) at I\ :sub:`E`\ =10 µA
+     - NPN forward Current Gain (\ :math:`\frac{I_C}{I_B})` at :math:`I_E`\ =10 µA
 
    * - BFNPN1X1\_1P0
      - 36.72
      - 17.97
      - 55.38
      - 
-     - NPN forward Current Gain (I\ :sub:`C`/I\ :sub:`B`) at I\ :sub:`E`\ =1.0 µA
+     - NPN forward Current Gain (\ :math:`\frac{I_C}{I_B})` at :math:`I_E`\ =1.0 µA
 
    * - BFNPN1X2\_17P5
      - 35.14
      - 16.98
      - 53.37
      - 
-     - NPN forward Current Gain (I\ :sub:`C`/I\ :sub:`B`) at I\ :sub:`E`\ =17.5 µA
+     - NPN forward Current Gain (\ :math:`\frac{I_C}{I_B})` at :math:`I_E`\ =17.5 µA
```

Comment:
Should the `=17.5 uA` be part of the math bit here?

Reply:
In recent commit rightside values and units are included in equation.